### DISCUSSION TITLE
氵

### DISCUSSION THREAD

**Discussion by Resister52** [share121/inter-knot#5006] [灌水]
原来左上角只是个图标么，我说怎么点了没反应，而且还有个臭臭的数字

**Comment by Mollsity** [comment]
想把那改了，未登录看着好烦😡👎

**Reply by manyshoes** [reply]
在哪登录

**Reply by manyshoes** [reply]
多谢![1726401750296e444c837d0aa581fce000f7f7def01339a6c6a979e42cd30119b61f661e035b0.0.jpg](https://github.com/user-attachments/assets/6b7846a7-62c6-4eff-abe6-184c8bcb1199)
